src/doc/style-guide/src/items.md

```diff
@@ -367,26 +367,49 @@ where
 ## Type aliases
 
 Keep type aliases on one line when they fit. If necessary to break the line, do
```

Comment:
@calebcartwright: Rustfmt doesn't try to keep type aliases on one line if there's a where clause. I guess this should be amended to say that.

Reply:
I think that may be implicitly covered in the prescriptions for where clauses in https://doc.rust-lang.org/nightly/style-guide/items.html#where-clauses, namely whether the `where` keyword needs a preceeding and subsequent linebreak

Reply:
I guess it depends on if "fit" means like.. "fits w.r.t. the line width" and also "is allowed to".

Reply:
Fair enough. Regarding single-line special cases, that'd need to wait 'til 2024 anyways, so I guess it's fine to keep this wordin :shrug:

Reply:
Can discuss at the meet that's about to start, but my recollection was that we previously decided we were going to have to punt this to 2024 too because of the amount of churn it would entail

Reply:
Decided in meeting that this will be implemented in the current v1 style edition.